src/WarcraftLegacies.Source/Quests/Warsong/QuestOrgrimmar.cs

```diff
@@ -30,14 +30,15 @@ public sealed class QuestOrgrimmar : QuestData
     }
 
     /// <inheritdoc/>
-    public override string RewardFlavour => "The city of Orgrimmar was finally constructed by the Warsong engineers, it is now a home for the new Horde and a symbol of power and innovation";
+    public override string RewardFlavour => "The city of Orgrimmar was finally constructed by the Warsong engineers, it is now a home for the new Horde and a symbol of power and innovation. The Warchief has rewarded us generously for our work!";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Control of all units in Orgrimmar, able to train Varok and Azerite Siege Engines";
+    protected override string RewardDescription => "Control of all units in Orgrimmar, 1200 gold, able to train Varok and Azerite Siege Engines";
```

Comment:
Use a constant here and on line 41 so that it's easier to maintain. See `QuestUpperNetherstorm.cs` for a good example of this.